### PR TITLE
changed default params for gemini

### PR DESCRIPTION
**Pull request by lkuligin**
Replace this entire comment with:
  - **Description:** changed default values for Vertex LLMs (to be handled on the SDK's side)

### CODE REVIEW

File: libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def test_candidates() -> None:
     assert len(response.generations[0]) == 2
 
 
-@pytest.mark.parametrize("model_name", ["chat-bison@001", "gemini-pro"])
+@pytest.mark.parametrize("model_name", ["chat-bison@001"])
```
Comment (efriis) [inline libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py:54]: Given the differences between the available params for palm and gemini, would it make sense to continue to test this?

**Review by efriis — COMMENTED** [review]
One comment, otherwise code lgtm.

Are the default params set the same in the sdk? I'm ok deferring to the sdk's default, and if they're different, it'd be good to message that this is a breaking change.

File: libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py (hunk 1 above)
Comment (lkuligin) [reply libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py:54]: I'd say "yes" for Palm.

**Comment by lkuligin** [comment]
> One comment, otherwise code lgtm.
> 
> Are the default params set the same in the sdk? I'm ok deferring to the sdk's default, and if they're different, it'd be good to message that this is a breaking change.

how should we do this? 

P.S. Gemini integration is relatively new, so maybe it's fine to introduce a breaking change, wdyt? There were a few complaints that chat model was throwing the exception, and the main reason is the low default `max_output_tokens` value (that I used the same as for Palm, and apparently it was not a good choice).

File: libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py (hunk 1 above)
Comment (efriis) [reply libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py:54]: Why not worth testing gemini here?

**Comment by efriis** [comment]
Breaking change on these defaults works for me.

The other option is if we're seeing divergence between Palm and Gemini features, we can introduce separate `ChatVertexGemini` and `VertexGemini` wrappers, and deprecate gemini usage through these wrappers. Thoughts?

File: libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py (hunk 1 above)
Comment (efriis) [reply libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py]: Sounds good. In the future, if the flakiness is something that should be fixed elsewhere, feel free to mark it as `@pytest.mark.xfail(...)`

**Review by efriis — APPROVED** [review]

**Pull request merged**